frontend/public/module/k8s/probe.ts

```diff
@@ -13,15 +14,15 @@ import {
 const HookAction = Object.freeze({
   exec: {
     id: 'exec',
-    label: 'Exec Command',
+    label: i18next.t('probe~Exec Command'),
   },
   httpGet: {
     id: 'httpGet',
-    label: 'HTTP Get',
+    label: i18next.t('probe~HTTP Get'),
```

Comment:
Not new, but the `GET` is an HTTP method and should be uppercase.

```suggestion
    label: i18next.t('probe~HTTP GET'),
```